### ISSUE TITLE
Convective adjustment fails if num_lat == num_lev

### ISSUE THREAD

**Issue by brian-rose** [brian-rose/climlab#5]
Code in `convadj.py` does a not-so-smart check for multi-dimensionality:

```
def Akamaev_adjustment_multidim(theta, q, beta, n_k, theta_k, s_k, t_k):
    L = q.size  # number of vertical levels
    size0 = theta.shape[0] #np.size(T, axis=0)
    if size0 != L:
```

This fails when you have a domain with same number of latitude points as vertical levels.

Workaround: don't use such a domain....

**Comment by rabernat** [comment]
In theory you could catch edge cases like this through testing. That's why #8 is important.

**Comment by brian-rose** [comment]
Finally fixed in #133
